src/navigation/mobileStepper/MobileStepper.jsx

```diff
@@ -0,0 +1,184 @@
+            display: 'inline-block',
+            height: 8,
+            marginRight: 5,
+            width: 8,
+        },
+        stepActive: {
+            animation: '$fadeIn 250ms ease-in-out',
+            background: palette.active.primary,
+        },
+        backButton: {
+            background: palette.grey[400],
+        },
+        nextButton: {
+            gridArea: 'nextButton',
+        },
+        '@keyframes fadeIn': {
+            '0%': {
+                opacity: 0,
+                transform: 'scale(0)',
+            },
+            '50%': {
+                opacity: 1,
+                transform: 'scale(1.4)',
+            },
+            '100%': {
+                opacity: 1,
+                transform: 'scale(1)',
+            },
+        },
+    }));
+
+    const classes = useStyles();
+
+    const backButton =
+        activeStep > 0 && steps - 1 ? (
+            <Button
+                className={classes.backButton}
+                id={`${id}--button_back`}
+                onClick={backStepAction}
+                width={90}
+            >
+                {backStepLabel}
+            </Button>
+        ) : null;
+
+    const lastButton =
+        activeStep === steps - 1 ? (
+            <Button
+                className={classes.nextButton}
+                color="success"
+                id={`${id}--button_last`}
+                onClick={lastStepAction}
+                width={90}
+            >
+                {lastStepLabel}
+            </Button>
+        ) : null;
+
+    const nextButton =
+        activeStep < steps - 1 ? (
+            <Button
+                className={classes.nextButton}
+                color="primary"
+                id={`${id}--button_next`}
+                onClick={nextStepAction}
+                width={activeStep === 0 ? 200 : 90}
+            >
+                {nextStepLabel}
+            </Button>
+        ) : null;
+
+    const callToActionButton =
+        !isEmpty(callToAction) && activeStep > 0 ? (
+            <Button
+                className={classes.callToAction}
+                color="outline"
+                id={`${id}--button_call_to_action`}
+                onClick={() => window.open(callToAction.url)}
+            >
+                {callToAction.text}
+            </Button>
+        ) : null;
+
+    const stepperIndicator = () => {
+        const stepper = [];
+        // eslint-disable-next-line no-plusplus
+        for (let i = 0; i < steps; i++) {
+            stepper.push(
+                <span
+                    className={`${classes.step} ${activeStep === i && classes.stepActive}`}
+                    key={`step${i}`}
+                />,
+            );
+        }
+        return stepper;
+    };
+
+    return (
+        steps > 0 && (
+            <div className={classes.mobileStepper}>
+                {backButton}
+                {callToActionButton}
+                {nextButton}
+                {lastButton}
```

Comment:
These buttons need to be optional. Much how MUI Mobile Stepper has theirs.

Reply:
Those are optional on the top lines (104, 116 and 129)